lib/railtie.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails'
+
+module Circuitry
+  class Railtie < Rails::Railtie
```

Comment:
Since this file is under the circuitry module, I think it belongs in `lib/circuitry` instead.